src/stream/src/from_proto/batch_query.rs

```diff
@@ -59,9 +68,9 @@ impl ExecutorBuilder for BatchQueryExecutorBuilder {
 
         // Use indices based on full table instead of streaming executor output.
         let pk_indices = table_desc
-            .pk_indices
+            .order_key
```

Comment:
Could you please enable `index.slt.disabled` to test whether order key is correct?

Reply:
failed 🥸
<img width="491" alt="image" src="https://user-images.githubusercontent.com/37948597/177277877-6b3b845a-ef3e-4aec-a44f-f458c238744a.png">

Reply:
At least it does not panic. 🤣